frontend/components/Task.tsx

```diff
@@ -3,24 +3,31 @@ import { useEffect } from "react";
 
 function Task(props: any): JSX.Element {
   interface ITask {
-    tasks: string[];
+    tasks: [id: number, title: string];
   }
 
   const taskList = async () => {
-    let allTasks = await axios.get<ITask>("tasks/");
+    let allTasks = await axios.get<ITask>("getTasks/");
     console.log(allTasks);
-    props.setter(allTasks.data.tasks);
+    props.setList(allTasks.data.tasks);
   };
 
+  useEffect(() => {
+    if (props.Submit) {
```

Comment:
I believe the naming convention is the first character should be lowercased unless it is a class.